### ISSUE TITLE
pizza_ya_san pizza guy and houndeyes not randomized

### ISSUE THREAD

**Issue by wootguy** [wootguy/Glitch-Life#21]
> it seems the pizza guys can only change to default army grunts and is the same with the houndeyes when they attack the shop owners. never had them change into something else from the number of times it has been fine.

**Comment by atmogenic** [comment]
another thing i forgot which u may have encountered is that the pizza owner on pizza ya san2 is a headcrab, just like with the pizza guys being grunts, the pizza owner always becomes a mudcrab

**Comment by atmogenic** [comment]
this bug is still present btw 21/03/21


https://mega.nz/file/JokHRQyZ#sli9DEYS2lDYelRvefsxgTC-WOSk1v1kaonF46hqM8A

This is the link to the map download  and more with the levels, its 34mb so i'm unable to attach it to github

**Comment by wootguy** [comment]
yeah haven't closed the issue yet. I lost all interest in this project so it might be left unfixed for a few more years. Right now I'm only making time for critical bugs like being unable to randomize _anything_ or having the program crash in a reproducible way.

**Comment by atmogenic** [comment]
oh ok
